### ISSUE TITLE
a11y - Modal Zoom Scaling

### ISSUE THREAD

**Issue by angelina-smith** [GSA/px-benefit-finder#1585]
### What's expected

As someone with low vision, I would expect all functionality, content, and the focus indicator to be viewable at 200-400% browser zoom.

### What happened

On Chrome, at 200% the modal's close button was obscured and not accessible with scrolling. When I keyboard tabbed to the close button, my focus indicator was not visible.
![image](https://github.com/user-attachments/assets/24945963-509a-417f-850b-d9fce71de278)

Furthermore, at 400%, more of the buttons were obscured and focus indicator was not fully visible around each button.
![image](https://github.com/user-attachments/assets/1d774433-1b4c-4aa1-80fe-56510edde572)

### How to replicate

Chrome at 200% and 400% browser zoom.

**Comment by scottqueen-bixal** [comment]
we need to shuffle some priorities on other work for some upcoming deadlines, putting this into backlog - will return to implement post 9/10
